# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Tino Kreutzer <tino.kreutzer@kobotoolbox.org>, 2016
# Tino Kreutzer <tino.kreutzer@kobotoolbox.org>, 2016
# TomonoriHoshi <tomonori.hoshi.japan@gmail.com>, 2022-2023
# 山田 敦士, 2023
# 山田 敦士, 2023
msgid ""
msgstr ""
"Project-Id-Version: KoboToolbox\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-02-09 19:00+0000\n"
"PO-Revision-Date: 2016-03-14 19:56+0000\n"
"Last-Translator: 山田 敦士, 2023\n"
"Language-Team: Japanese (http://app.transifex.com/kobotoolbox/kobotoolbox/language/ja/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ja\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: hub/templates/admin/loginas/change_form.html:7
msgid "Log in as user"
msgstr "ユーザーとしてログインする"

#: kobo/apps/accounts/forms.py:68
msgid "Please enter a valid URL"
msgstr "有効なURLを入力してください"

#: kobo/apps/accounts/forms.py:76
msgid "Non-profit organization"
msgstr "非営利団体（NPO）"

#: kobo/apps/accounts/forms.py:77
msgid "Government institution"
msgstr "政府機関"

#: kobo/apps/accounts/forms.py:78
msgid "Educational organization"
msgstr "教育組織"

#: kobo/apps/accounts/forms.py:79
msgid "A commercial/for-profit company"
msgstr "営利目的の企業"

#: kobo/apps/accounts/forms.py:80
msgid "I am not associated with any organization"
msgstr "私はいかなる組織にも属していません"

#: kobo/apps/accounts/forms.py:88
msgid "Male"
msgstr "男性"

#: kobo/apps/accounts/forms.py:89
msgid "Female"
msgstr "女性"

#: kobo/apps/accounts/forms.py:90 kobo/static_lists.py:37
msgid "Other"
msgstr "その他"

#: kobo/apps/accounts/forms.py:124 kpi/templates/base.html:29
msgid "Terms of Service"
msgstr "サービスの利用規約"

#: kobo/apps/accounts/forms.py:131 kpi/templates/base.html:34
msgid "Privacy Policy"
msgstr "プライバシーポリシー"

#: kobo/apps/accounts/forms.py:133
msgid "I agree with the ##terms_of_service## and ##privacy_policy##"
msgstr "##terms_of_service##と##privacy_policy##に同意します"

#: kobo/apps/accounts/forms.py:143
msgid "Password confirmation"
msgstr "パスワード認証"

#: kobo/apps/accounts/forms.py:145
msgid "name@organization.org"
msgstr "name@organization.org"

#: kobo/apps/accounts/forms.py:220
msgid "This field is required."
msgstr "この項目は必須です。"

#: kobo/apps/accounts/forms.py:307
msgid "You must type the same password each time."
msgstr "いつも同じパスワードを入力する必要があります。"

#: kobo/apps/accounts/mfa/forms.py:57
msgid "Enter the ##token length##-character token"
msgstr "##token length##-文字のトークンを挿入してください"

#: kobo/apps/accounts/mfa/forms.py:69
msgid "Your token is invalid"
msgstr "トークンが無効です"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:21
msgid "Please enter your verification token"
msgstr "認証トークンを入力してください"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:25
msgid ""
"Please enter the verification token displayed by your authenticator app."
msgstr "認証アプリに表示されている認証トークンを入力してください。"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:32
msgid "Problems with the token"
msgstr "トークンに問題があります"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:37
msgid "Continue"
msgstr "続行"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:45
msgid "Verification issues"
msgstr "検証に問題があります"

#: kobo/apps/accounts/mfa/templates/mfa_token.html:53
msgid "Back"
msgstr "戻る"

#: kobo/apps/accounts/templates/account/email/email_confirmation_message.txt:3
msgid ""
"To confirm the new email address associated with your KoboToolbox account, "
"please follow the link below."
msgstr "KoboToolboxアカウントに関連付けられた新しいメールアドレスは、以下のリンクからご確認ください。"

#: kobo/apps/accounts/templates/account/email/email_confirmation_message.txt:7
msgid ""
"This link will expire in one hour. To request a new verification link, "
"please go to your account settings and re-enter your new email address:"
msgstr "このリンクは1時間後に失効します。新しい認証リンクをリクエストするには、アカウント設定に移動して、新しいメールアドレスを再入力してください:"

#: kobo/apps/accounts/templates/account/email/email_confirmation_message.txt:10
#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:29
#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:15
#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:15
#: kobo/apps/hook/templates/reports/failures_email_body.html:37
#: kobo/apps/hook/templates/reports/failures_email_body.txt:29
msgid "Best,"
msgstr "敬具"

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:10
msgid "Thanks for signing up with KoboToolbox!"
msgstr "KoboToolboxへの登録ありがとうございます！"

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:12
msgid ""
"Confirming your account will give you access to KoboToolbox applications. "
"Please visit the following URL to finish activation of your new account."
msgstr "アカウントを確認すると、KoboToolboxのアプリケーションにアクセスできるようになります。以下のURLにアクセスし、新しいアカウントのアクティベーションを完了してください。"

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:16
msgid "Your username is: "
msgstr "ユーザーネーム： "

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:25
msgid ""
"For help getting started, check out the KoboToolbox user documentation: "
"https://support.kobotoolbox.com "
msgstr "ご使用を開始いただくためのヘルプについては、、KoboToolbox<b> </b>ユーザードキュメントをご覧ください:https://support.kobotoolbox.com "

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_message.txt:27
msgid ""
"You can also join the KoboToolbox community forum to ask questions, share "
"solutions, and chat with thousands of users: "
"https://community.kobotoolbox.org"
msgstr "KoboToolboxのコミュニティフォーラムに参加して、質問したり、解決策を共有したり、何千人ものユーザーとチャットすることもできます:https://community.kobotoolbox.org"

#: kobo/apps/accounts/templates/account/email/email_confirmation_signup_subject.txt:10
msgid "Activate your KoboToolbox Account"
msgstr "KoboToolboxアカウントをアクティベートする"

#: kobo/apps/accounts/templates/account/email/email_confirmation_subject.txt:3
msgid "KoboToolbox account email address verification"
msgstr "KoboToolboxアカウントのメールアドレス認証"

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:4
#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:5
msgid "Hello from KoboToolbox,"
msgstr "KoboToolboxより、こんにちは、"

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:6
msgid "We’ve received a password reset request for your user account."
msgstr "ユーザーアカウントからパスワードリセットのリクエストを受け取りました。"

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:8
msgid "To reset your password, click the link below."
msgstr "パスワードをリセットするには、以下のリンクをクリックしてください。"

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:11
#, python-format
msgid "Your username is %(username)s."
msgstr "あなたのユーザーネームは%(username)sです。"

#: kobo/apps/accounts/templates/account/email/password_reset_key_message.txt:13
#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:9
msgid ""
"If you did not request a password reset, you can safely ignore this email."
msgstr "パスワードのリセットをリクエストしていない場合は、このメールを無視していただいて結構です。"

#: kobo/apps/accounts/templates/account/email/password_reset_key_subject.txt:3
msgid "KoboToolbox Password Reset"
msgstr "KoboToolboxのパスワード再設定"

#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:6
msgid ""
"You are receiving this email because you or someone else has requested a "
"password for their KoboToolbox account."
msgstr "このメールを受信しているのは、ご自身、または他の誰かが、KoboToolboxアカウントのパスワードを要求したためです。"

#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:7
#, python-format
msgid ""
"However, we do not have any record of an account with email %(email)s in our"
" database."
msgstr "しかしながら、データベースには、メールアドレス%(email)sを使用したアカウントの記録はありません。"

#: kobo/apps/accounts/templates/account/email/unknown_account_message.txt:11
msgid ""
"If it was you, please try another email address you might have used to "
"create your account, or double check that your account was not registered on"
" a different KoboToolbox server."
msgstr "ご自身である場合は、アカウント作成に使用した可能性のある別のメールアドレスを試すか、アカウントが別のKoboToolboxサーバーに登録されていないか、再度確認してください。"

#: kobo/apps/accounts/templates/account/email/unknown_account_subject.txt:3
msgid "KoboToolbox Password Reset Unsuccessful"
msgstr "KoboToolboxのパスワードリセットに失敗しました"

#: kobo/apps/accounts/templates/account/email_confirm.html:15
msgid "Confirm E-mail Address"
msgstr "メールアドレスを確認してください"

#: kobo/apps/accounts/templates/account/email_confirm.html:19
#, python-format
msgid ""
"Please confirm that <a href=\"mailto:%(email)s\">%(email)s</a> is an e-mail "
"address for user %(user_display)s."
msgstr "<a href=\"mailto:%(email)s\">%(email)s</a>がユーザー%(user_display)sのメールアドレスであることを確認してください。"

#: kobo/apps/accounts/templates/account/email_confirm.html:26
msgid "Confirm"
msgstr "確認"

#: kobo/apps/accounts/templates/account/email_confirm.html:30
msgid "Activation failed"
msgstr "アクティベーション失敗"

#: kobo/apps/accounts/templates/account/email_confirm.html:33
msgid ""
"Sorry, we couldn't activate your account. The link we sent you has expired."
msgstr "申し訳ございませんが、アカウントを有効にすることができませんでした。送信したリンクの有効期限が切れています。"

#: kobo/apps/accounts/templates/account/email_confirm.html:39
msgid ""
"This e-mail confirmation link expired or is invalid.\n"
"      If you need to request a new link to verify your email address change,\n"
"      please go to your account settings page and enter your email address again."
msgstr "このメールアドレス確認リンクは有効期限が切れているか、無効になっています。\n　メールアドレスの変更を確認するために、新しいリンクをリクエストする必要がある場合は、\n　アカウント設定ページに移動してメールアドレスを再入力してください。"

#: kobo/apps/accounts/templates/account/login.html:29
msgid "Login"
msgstr "ログイン"

#: kobo/apps/accounts/templates/account/login.html:36
#: kobo/apps/accounts/templates/account/signup.html:14
msgid "Create an account"
msgstr "アカウントの作成"

#: kobo/apps/accounts/templates/account/login.html:41
msgid "Forgot password?"
msgstr "パスワードをお忘れですか？"

#: kobo/apps/accounts/templates/account/login.html:50
msgid "Log in using SSO"
msgstr "SSOでログイン"

#: kobo/apps/accounts/templates/account/login.html:52
#: kobo/apps/accounts/templates/account/signup.html:63
msgid ""
"If you set up Single Sign On in your account settings, you will only be able"
" to log in through this method."
msgstr "アカウント設定でシングルサインオンを設定すると、この方法でのみログインできるようになります。"

#: kobo/apps/accounts/templates/account/password_reset.html:10
msgid "Password Reset"
msgstr "パスワードのリセット"

#: kobo/apps/accounts/templates/account/password_reset.html:19
msgid "Reset Password"
msgstr "パスワードをリセットする"

#: kobo/apps/accounts/templates/account/password_reset_complete.html:11
#: kobo/apps/accounts/templates/account/password_reset_from_key_done.html:11
msgid "Password reset successfully"
msgstr "パスワードのリセット成功"

#: kobo/apps/accounts/templates/account/password_reset_complete.html:14
#: kobo/apps/accounts/templates/account/password_reset_from_key_done.html:14
#: kobo/apps/accounts/templates/socialaccount/login.html:34
msgid "Log in"
msgstr "ログイン"

#: kobo/apps/accounts/templates/account/password_reset_done.html:12
msgid ""
"Your request has been received. If the email address you entered corresponds"
" to an account on this service, a message has been sent with password reset "
"instructions. If you do not receive an email, your account might be "
"registered under a different address, or you might have entered your address"
" incorrectly."
msgstr "リクエストを受理しました。入力したeメールアドレスが本サービス上のアカウントに該当する場合、パスワード再設定の方法が記されたメールが送られます。メールが届かない場合は、異なるアドレスで登録している、または入力したアドレスに間違いがある可能性があります。"

#: kobo/apps/accounts/templates/account/password_reset_from_key.html:13
msgid "Password reset failed"
msgstr "パスワード再設定に失敗しました"

#: kobo/apps/accounts/templates/account/password_reset_from_key.html:25
msgid "Submit"
msgstr "提出"

#: kobo/apps/accounts/templates/account/signup.html:47
msgid "Create Account"
msgstr "アカウントを作成"

#: kobo/apps/accounts/templates/account/signup.html:51
#: kobo/apps/accounts/templates/socialaccount/login.html:39
msgid "or"
msgstr "または"

#: kobo/apps/accounts/templates/account/signup.html:51
msgid "login"
msgstr "ログイン"

#: kobo/apps/accounts/templates/account/signup.html:61
msgid "Register using SSO"
msgstr "SSOで登録"

#: kobo/apps/accounts/templates/account/signup.html:75
msgid "Register with SSO"
msgstr "SSOで登録"

#: kobo/apps/accounts/templates/account/verification_sent.html:8
msgid "Confirm your email address"
msgstr "メールアドレスを確認"

#: kobo/apps/accounts/templates/account/verification_sent.html:11
msgid "Please click the activation link in the email just sent to you."
msgstr "先ほど送信されたメールに記載されているアクティベーションリンクをクリックしてください。"

#: kobo/apps/accounts/templates/socialaccount/login.html:14
msgid "Set up Single Sign On"
msgstr "シングルサインオンを設定する"

#: kobo/apps/accounts/templates/socialaccount/login.html:16
#, python-format
msgid ""
"Click the button below to connect your KoboToolbox account with your "
"%(appname)s account."
msgstr "以下のボタンをクリックして、KoboToolbox アカウントと%(appname)sのアカウントを接続します。"

#: kobo/apps/accounts/templates/socialaccount/login.html:18
msgid "This will allow you to use single sign-on (SSO) in the future."
msgstr "これにより将来的にシングルサインオン（SSO）を利用することができるようになります。"

#: kobo/apps/accounts/templates/socialaccount/login.html:23
#, python-format
msgid "Connect with %(appname)s"
msgstr "%(appname)sと接続"

#: kobo/apps/accounts/templates/socialaccount/login.html:27
msgid "Log in with "
msgstr "でログインする "

#: kobo/apps/accounts/templates/socialaccount/login.html:29
#, python-format
msgid "Log in to KoboToolbox with your %(appname)s account."
msgstr "%(appname)sアカウントでKoboToolboxにログインする."

#: kobo/apps/accounts/templates/socialaccount/login.html:39
msgid "go back"
msgstr "戻る"

#: kobo/apps/accounts/templates/socialaccount/signup.html:11
#, python-format
msgid "Welcome to KoboToolbox, %(username)s!"
msgstr "KoboToolboxへようこそ、%(username)s！"

#: kobo/apps/accounts/templates/socialaccount/signup.html:13
msgid "Welcome to KoboToolbox!"
msgstr "KoboToolboxへようこそ！"

#: kobo/apps/accounts/templates/socialaccount/signup.html:18
#, python-format
msgid "You are about to use your %(appname)s account to login to KoboToolbox."
msgstr "KoboToolboxにログインするために、%(appname)sのアカウントを使用しようとしています。"

#: kobo/apps/accounts/templates/socialaccount/signup.html:18
msgid "As a final step, please complete the following form"
msgstr "最後に、以下のフォームにご記入ください"

#: kobo/apps/accounts/templates/socialaccount/signup.html:44
msgid "Save and Access"
msgstr "保存してアクセス"

#: kobo/apps/accounts/validators.py:7
msgid ""
"Usernames must be between 2 and 30 characters in length, and may only "
"consist of lowercase letters, numbers, and underscores, where the first "
"character must be a letter."
msgstr "ユーザーネームは2～30文字数で、小文字、数字、アンダーバーで構成され、冒頭は文字である必要があります。"

#: kobo/apps/hook/serializers/v2/hook.py:46
msgid "Invalid scheme"
msgstr "無効な組み合わせです"

#: kobo/apps/hook/serializers/v2/hook.py:49
msgid "Unsecured endpoint is not allowed"
msgstr "セキュリティーで保護されていないエンドポイントは許可されません"

#: kobo/apps/hook/serializers/v2/hook.py:60
msgid "Invalid JSON"
msgstr "無効なＪＳＯＮです"

#: kobo/apps/hook/serializers/v2/hook.py:70
msgid "Can be used only with JSON submission format"
msgstr "JSON送信フォーマットでのみ使用可能です"

#: kobo/apps/hook/tasks.py:148
msgid "REST Services Failure Report"
msgstr "RESTサービスのエラーレポート"

#: kobo/apps/hook/templates/reports/failures_email_body.html:3
#: kobo/apps/hook/templates/reports/failures_email_body.txt:3
msgid "Project name"
msgstr "プロジェクト名"

#: kobo/apps/hook/templates/reports/failures_email_body.html:4
#: kobo/apps/hook/templates/reports/failures_email_body.txt:4
msgid "Hook"
msgstr "フック"

#: kobo/apps/hook/templates/reports/failures_email_body.html:5
#: kobo/apps/hook/templates/reports/failures_email_body.txt:5
msgid "Submission"
msgstr "投稿"

#: kobo/apps/hook/templates/reports/failures_email_body.html:6
#: kobo/apps/hook/templates/reports/failures_email_body.txt:6
msgid "Status code"
msgstr "ステータスコード"

#: kobo/apps/hook/templates/reports/failures_email_body.html:7
#: kobo/apps/hook/templates/reports/failures_email_body.txt:7
msgid "Message"
msgstr "メッセージ"

#: kobo/apps/hook/templates/reports/failures_email_body.html:8
#: kobo/apps/hook/templates/reports/failures_email_body.txt:8
msgid "Date"
msgstr "日付"

#: kobo/apps/hook/templates/reports/failures_email_body.html:10
#: kobo/apps/hook/templates/reports/failures_email_body.txt:10
msgid "Dear"
msgstr "様"

#: kobo/apps/hook/templates/reports/failures_email_body.html:11
#: kobo/apps/hook/templates/reports/failures_email_body.txt:12
msgid "Some submissions could not be sent to their external endpoints."
msgstr "一部の提出フォームが外部エンドポイントに送信できませんでした。"

#: kobo/apps/hook/views/v2/hook.py:174
msgid "Task successfully scheduled"
msgstr "タスクは正常にスケジュールされました"

#: kobo/apps/hook/views/v2/hook.py:203
msgid "No data to retry"
msgstr "リトライするデータはありません"

#: kobo/apps/hook/views/v2/hook.py:206
msgid "Can not retry on disabled hooks"
msgstr "無効にされたフックでリトライできません"

#: kobo/apps/hook/views/v2/hook_log.py:121
msgid "An error has occurred when sending the data. Please try again later."
msgstr "データ送信でエラーが発生しました。後ほどもう一度トライしてください。"

#: kobo/apps/hook/views/v2/hook_log.py:125
msgid "Data is being or has already been processed"
msgstr "データは処理中か既に処理されています"

#: kobo/apps/hook/views/v2/hook_signal.py:54
#: kpi/deployment_backends/base_backend.py:593
#: kpi/deployment_backends/base_backend.py:600
msgid "A positive integer is required."
msgstr "正の整数が必須です。"

#: kobo/apps/hook/views/v2/hook_signal.py:71
msgid ""
"We got and saved your data, but may not have fully processed it. You should "
"not try to resubmit."
msgstr "データを受け取り保存しましたが、完全に処理できていない可能性があります。再提出を試みないでください。"

#: kobo/apps/hook/views/v2/hook_signal.py:80
msgid "Your data for instance {} has been already submitted."
msgstr "例えば{}のデータはすでに送信されています。"

#: kobo/apps/languages/templates/admin/languages_change_list.html:8
msgid "Import"
msgstr "インポート"

#: kobo/apps/reports/report_data.py:136
msgid "`{}` not found."
msgstr "`{}`が見つかりません。"

#: kobo/apps/reports/report_data.py:141
msgid "`{}` is not a select one question."
msgstr "`{}`は質問を1つ選択していません。"

#: kobo/settings/base.py:245
msgid ""
"If you cannot access your authenticator app, please enter one of your backup"
" codes instead. If you cannot access those either, then you will need to "
"request assistance by contacting [##support email##](mailto:##support "
"email##)."
msgstr "認証アプリケーションにアクセスできない時は、代わりにバックアップコードの1つを入力してください。そのどちらにもアクセスできない時は、［##support email##］（mailto:##support email##）で支援を求める必要があります。"

#: kobo/settings/base.py:446
msgid ""
"The password must be at least 10 characters long and contain 3 or more of "
"the following: uppercase letters, lowercase letters, numbers, and special "
"characters. It cannot be similar to your name, username, or email address."
msgstr "パスワードは10文字以上の長さで、大文字、小文字、数字、特殊記号のうち3つ以上含んでいる必要があります。名前や、ユーザーネーム、eメールアドレス類似は使えません。"

#: kobo/static_lists.py:13
msgid "Public Administration"
msgstr "公的機関"

#: kobo/static_lists.py:14
msgid "Arts, Entertainment, and Recreation"
msgstr "芸術、エンターテインメント、レクレーション"

#: kobo/static_lists.py:15
msgid "Educational Services / Higher Education"
msgstr "教育サービス／高等教育"

#: kobo/static_lists.py:16
msgid "Health Services / Public Health"
msgstr "ヘルスサービス／公衆衛生"

#: kobo/static_lists.py:17
msgid "Finance and Insurance"
msgstr "金融と保険"

#: kobo/static_lists.py:18
msgid "Information / Media"
msgstr "情報／メディア"

#: kobo/static_lists.py:19
msgid "Economic/Social Development"
msgstr "経済／社会発展"

#: kobo/static_lists.py:20
msgid "Security / Police / Peacekeeping"
msgstr "防犯／警察／平和維持"

#: kobo/static_lists.py:21
msgid "Disarmament & Demobilization"
msgstr "軍備縮小と招集解除"

#: kobo/static_lists.py:22
msgid "Environment"
msgstr "環境"

#: kobo/static_lists.py:23
msgid "Private sector"
msgstr "民間部門"

#: kobo/static_lists.py:24
msgid "Humanitarian - Coordination / Information Management"
msgstr "人道ー連携／情報管理"

#: kobo/static_lists.py:25
msgid "Humanitarian - Multiple Clusters"
msgstr "人道ー複数クラスター"

#: kobo/static_lists.py:26
msgid "Humanitarian - Camp Management & Coordination"
msgstr "人道ーキャンプ管理と連携"

#: kobo/static_lists.py:27
msgid "Humanitarian - Early Recovery"
msgstr "人道ー早期回復"

#: kobo/static_lists.py:28
msgid "Humanitarian - Education"
msgstr "人道ー教育"

#: kobo/static_lists.py:29
msgid "Humanitarian - Emergency Shelter"
msgstr "人道ー緊急シェルター"

#: kobo/static_lists.py:30
msgid "Humanitarian - Emergency Telecoms"
msgstr "人道―緊急通信"

#: kobo/static_lists.py:31
msgid "Humanitarian - Food Security"
msgstr "人道ー食糧保全"

#: kobo/static_lists.py:32
msgid "Humanitarian - Health"
msgstr "人道ー健康"

#: kobo/static_lists.py:33
msgid "Humanitarian - Logistics"
msgstr "人道ー輸送"

#: kobo/static_lists.py:34
msgid "Humanitarian - Nutrition"
msgstr "人道―栄養"

#: kobo/static_lists.py:35
msgid "Humanitarian - Protection"
msgstr "人道ープロテクション"

#: kobo/static_lists.py:36
msgid "Humanitarian - Sanitation, Water & Hygiene"
msgstr "人道ー公衆衛生、水、保健"

#: kobo/static_lists.py:344 kobo/static_lists.py:356
msgid "Sector"
msgstr "セクター"

#: kobo/static_lists.py:345 kobo/static_lists.py:360
msgid "Country"
msgstr "国"

#: kobo/static_lists.py:346
msgid "Operational purpose of data"
msgstr "データの利用目的"

#: kobo/static_lists.py:347
msgid "Does this project collect personally identifiable information?"
msgstr "このプロジェクトは個人を特定できる情報を収集しますか？"

#: kobo/static_lists.py:348
msgid "Description"
msgstr "詳細"

#: kobo/static_lists.py:352 kpi/password_validation.py:109
msgid "Full name"
msgstr "フルネーム"

#: kobo/static_lists.py:353 kpi/password_validation.py:110
msgid "Organization name"
msgstr "組織名"

#: kobo/static_lists.py:354
msgid "Organization type"
msgstr "組織のタイプ"

#: kobo/static_lists.py:355
msgid "Organization website"
msgstr "組織のウェブサイト"

#: kobo/static_lists.py:357
msgid "Gender"
msgstr "性別"

#: kobo/static_lists.py:358
msgid "Bio"
msgstr "プロフィール"

#: kobo/static_lists.py:359
msgid "City"
msgstr "市"

#: kobo/static_lists.py:361
msgid "Twitter"
msgstr "Twitter"

#: kobo/static_lists.py:362
msgid "LinkedIn"
msgstr "LinkedIn"

#: kobo/static_lists.py:363
msgid "Instagram"
msgstr "インスタグラム"

#: kobo/static_lists.py:364
msgid "I want to receive occasional updates about KoboToolbox"
msgstr "KoboToolboxに関する最新情報を受け取ることを希望します"

#: kpi/authentication.py:59
msgid "Invalid username/password"
msgstr "無効なユーザーネーム／パスワード"

#: kpi/deployment_backends/base_backend.py:122
msgid "No submissions match the given `submission_ids`"
msgstr "指定された`submission_ids`に一致する投稿はありません"

#: kpi/deployment_backends/base_backend.py:522
msgid ""
"This param is not implemented. Use `count` property of the response instead."
msgstr "このparamは実装されていません。代わりに返事の`count`プロパティーを使用してください。"

#: kpi/deployment_backends/base_backend.py:533
msgid "This param is not supported in `XML` format"
msgstr "このparamは`XML`フォーマットでサポートされていません"

#: kpi/deployment_backends/base_backend.py:538
msgid "This is not supported in `XML` format"
msgstr "これは`XML`フォーマットでサポートされていません"

#: kpi/deployment_backends/base_backend.py:558
#: kpi/deployment_backends/base_backend.py:586
#: kpi/deployment_backends/base_backend.py:608 kpi/views/v2/data.py:523
msgid "Value must be valid JSON."
msgstr "値は有効なJSONでなければなりません。"

#: kpi/deployment_backends/base_backend.py:563
msgid "Value must be a list."
msgstr "値はリストでなければなりません。"

#: kpi/deployment_backends/kobocat_backend.py:426
msgid "Your submission XML is malformed."
msgstr "提出されたXMLの形式が不正です。"

#: kpi/deployment_backends/kobocat_backend.py:435
msgid "Your submission XML is missing critical elements."
msgstr "提出されたXＭＬには重要な要素が欠けています。"

#: kpi/deployment_backends/kobocat_backend.py:448
msgid ""
"The submission you attempted to edit could not be found, or you do not have "
"access to it."
msgstr "編集を試みた投稿は見つかりません。あるいはアクセスできません。"

#: kpi/deployment_backends/kobocat_backend.py:1529
msgid "KoBoCAT returned an unexpected response: {}"
msgstr "ＫｏBoCATは予期しないレスポンス返しました：｛｝"

#: kpi/deployment_backends/kobocat_backend.py:1565
msgid "Something went wrong"
msgstr "問題が発生しました"

#: kpi/exceptions.py:49
msgid "Invalid payload for bulk updating of submissions"
msgstr "投稿の一括アップデートのペイロードが無効です"

#: kpi/exceptions.py:64
msgid "Must call `asset.connect_deployment()` first"
msgstr "最初に`asset.connect_deployment()`を呼ぶ必要があります"

#: kpi/exceptions.py:80
msgid ""
"Your access is restricted. Please reclaim your access by changing your "
"password at ##koboform_url##/accounts/password/reset/."
msgstr "アクセスは制限されています。##koboform_url##、アカウント、パスワード／リセットでパスワードを変更しアクセスを回復してください。"

#: kpi/exceptions.py:89
msgid "Invalid search. Please try again"
msgstr "無効な検索です。もう一度試してください"

#: kpi/exceptions.py:129
msgid "An error occurred trying to duplicate the submission"
msgstr "投稿の複製しようとしてエラーが発生しました"

#: kpi/exceptions.py:143
msgid "The specified object has not been deployed"
msgstr "指定のオブジェクトが配置されていません"

#: kpi/exceptions.py:152
msgid "Bad syntax"
msgstr "悪い構文です"

#: kpi/exceptions.py:157
msgid "Not supported field lookup"
msgstr "サポートされていないフィールドを検索"

#: kpi/exceptions.py:168
msgid "Your query is too short"
msgstr "クエリが短すぎます"

#: kpi/fields/jsonschema_form_field.py:23
#: kpi/fields/jsonschema_form_field.py:26
msgid "Enter valid JSON."
msgstr "有効なJSONを入力。"

#: kpi/fields/jsonschema_form_field.py:158
msgid "`##place_holder##` field cannot be hidden."
msgstr "`##place_holder##`フィールドは非表示にできません。"

#: kpi/fields/writable_json.py:30
#, python-brace-format
msgid "Unable to parse JSON: {error}"
msgstr "JSONを解析できません：{error}"

#: kpi/mixins/mfa.py:24
msgid "Multi-factor authentication is enabled for this account. "
msgstr "このアカウントでは多要素認証が有効です。 "

#: kpi/models/asset.py:251
msgid "Can view asset"
msgstr "アセットを閲覧できます"

#: kpi/models/asset.py:252
msgid "Can discover asset in public lists"
msgstr "公開リストにあるアセットを見つけられます"

#: kpi/models/asset.py:253
msgid "Can manage all aspects of asset"
msgstr "アセットのあらゆる側面を管理できます"

#: kpi/models/asset.py:255
msgid "Can submit data to asset"
msgstr "データをアセットに投稿できます"

#: kpi/models/asset.py:256
msgid "Can view submitted data for asset"
msgstr "アセットに投稿されたデータを見ることができます"

#: kpi/models/asset.py:257
msgid ""
"Can make partial actions on submitted data for asset for specific users"
msgstr "アセットに提出されたデータに対して、特定のユーザーのために部分的なアクションを行うことができる"

#: kpi/models/asset.py:260
msgid "Can modify submitted data for asset"
msgstr "アセットに提出されたデータを修正できます"

#: kpi/models/asset.py:261
msgid "Can delete submitted data for asset"
msgstr "アセットに提出されたデータを削除できます"

#: kpi/models/asset.py:262
msgid "Can validate submitted data asset"
msgstr "アセットに提出されたデータを有効にします"

#: kpi/models/asset.py:285
msgid "project"
msgstr "プロジェクト"

#: kpi/models/asset.py:285
msgid "form"
msgstr "フォーム"

#: kpi/models/asset.py:287
msgid "template"
msgstr "テンプレート"

#: kpi/models/asset.py:288
msgid "block"
msgstr "ブロック"

#: kpi/models/asset.py:289
msgid "question"
msgstr "質問"

#: kpi/models/asset.py:290
msgid "text"
msgstr "テキスト"

#: kpi/models/asset.py:291
msgid "empty"
msgstr "空"

#: kpi/models/asset.py:292
msgid "collection"
msgstr "コレクション"

#: kpi/models/asset.py:299
msgid "View ##asset_type_label##"
msgstr "閲覧##asset_type_label##"

#: kpi/models/asset.py:300
msgid "Edit ##asset_type_label##"
msgstr "編集##asset_type_label##"

#: kpi/models/asset.py:301
msgid "Discover ##asset_type_label##"
msgstr "発見##asset_type_label##"

#: kpi/models/asset.py:302
msgid "Manage ##asset_type_label##"
msgstr "管理##asset_type_label##"

#: kpi/models/asset.py:303
msgid "Add submissions"
msgstr "投稿を追加"

#: kpi/models/asset.py:304
msgid "View submissions"
msgstr "投稿を見る"

#: kpi/models/asset.py:307
msgid "Act on submissions only from specific users"
msgstr "特定のユーザーだけの投稿に対応"

#: kpi/models/asset.py:310
msgid "View submissions only from specific users"
msgstr "特定のユーザーだけの投稿を見る"

#: kpi/models/asset.py:313
msgid "Edit submissions only from specific users"
msgstr "特定のユーザーからの投稿を編集"

#: kpi/models/asset.py:316
msgid "Delete submissions only from specific users"
msgstr "特定のユーザーからの投稿を削除"

#: kpi/models/asset.py:319
msgid "Validate submissions only from specific users"
msgstr "特定のユーザーからの投稿を有効にする"

#: kpi/models/asset.py:322
msgid "Edit submissions"
msgstr "投稿を編集する"

#: kpi/models/asset.py:323
msgid "Delete submissions"
msgstr "投稿を削除する"

#: kpi/models/asset.py:324
msgid "Validate submissions"
msgstr "投稿を有効にする"

#: kpi/models/asset.py:629
msgid "Only partial permissions for submissions are supported"
msgstr "投稿の部分的な許可のみサポート"

#: kpi/models/asset.py:1331
msgid "Can not assign '{}' permission to owner"
msgstr "所有者に'{}'パーミッションを割り当てられません"

#: kpi/models/asset.py:1335
msgid "Can not assign '{}' permission. Partial permissions are missing."
msgstr "パーミッション '{}' を割り当てられません。パーミッションの一部が欠落しています。"

#: kpi/models/authorized_application.py:40
msgid "Invalid token."
msgstr "無効なトークン。"

#: kpi/models/import_export_task.py:142
msgid "Cannot access data"
msgstr "データにアクセスできません"

#: kpi/models/import_export_task.py:578
msgid "This data does not exist or you do not have access to it"
msgstr "このデータは存在しないか、アクセスできません"

#: kpi/password_validation.py:95
msgid "You cannot reuse your last password."
msgstr "最後のパスワードを再利用できません。"

#: kpi/password_validation.py:103
msgid "Your password cannot be the same as your previous one."
msgstr "以前と同一のパスワードは使えません。"

#: kpi/password_validation.py:170
msgid ""
"The password must contain at least ##number of rules## different kinds of "
"characters."
msgstr "パスワードには少なくとも##number of rules##種類以上の文字を含めなければなりません。"

#: kpi/serializers/current_user.py:162
msgid "Incorrect current password."
msgstr "現在のパスワードが間違っています。"

#: kpi/serializers/current_user.py:182
msgid "`current_password` and `new_password` must both be"
msgstr "`current_password`と`new_password`の両方がなければなりません"

#: kpi/serializers/current_user.py:222
#: kpi/serializers/v2/asset_permission_assignment.py:290
msgid "This field may not be blank."
msgstr "このフィールドは空白ではいけません"

#: kpi/serializers/v2/asset.py:130
#, python-format
msgid "%(count)d project has been undeleted"
msgstr "%(count)dプロジェクトは削除されませんでした"

#: kpi/serializers/v2/asset.py:136
#, python-format
msgid "%(count)d project has been deleted"
msgstr "%(count)dプロジェクトは削除されています"

#: kpi/serializers/v2/asset.py:143
#, python-format
msgid "%(count)d project has been unarchived"
msgstr "%(count)dプロジェクトはアーカイブされていません"

#: kpi/serializers/v2/asset.py:149
#, python-format
msgid "%(count)d project has been archived"
msgstr "%(count)dプロジェクトはアーカイブされています"

#: kpi/serializers/v2/asset.py:165
msgid "One or many projects have been deleted already!"
msgstr "一つ以上のプロジェクトが既に削除されています！"

#: kpi/serializers/v2/asset.py:173
msgid "One or many projects are already being deleted!"
msgstr "一つ以上のプロジェクトが既に削除されています！"

#: kpi/serializers/v2/asset.py:246
msgid "`action` parameter is required"
msgstr "`action`パラメーターは必須です"

#: kpi/serializers/v2/asset.py:251
msgid "Supported values for `action` are: "
msgstr "`action`がサポートするバリューは： "

#: kpi/serializers/v2/asset.py:273
msgid "Draft projects cannot be archived"
msgstr "プロジェクトの草稿はアーカイブできません"

#: kpi/serializers/v2/asset.py:279 kpi/serializers/v2/data.py:61
msgid "Confirmation is required"
msgstr "確認は必須です"

#: kpi/serializers/v2/asset.py:786
msgid "The property is required"
msgstr "プロパティーは必須です"

#: kpi/serializers/v2/asset.py:790
msgid "The property must be an array"
msgstr "プロパティーはアレーである必要があります"

#: kpi/serializers/v2/asset.py:812
#, python-brace-format
msgid "Some fields are invalid, choices are: `{valid_fields}`"
msgstr "無効なフィールドがあります。選択は`{valid_fields}`"

#: kpi/serializers/v2/asset.py:834
msgid "User cannot update current parent collection"
msgstr "ユーザーは現在の親コレクションをアップデートできません"

#: kpi/serializers/v2/asset.py:844
#: kpi/tests/api/v2/test_api_collections.py:482
msgid "Target collection not found"
msgstr "対象のコレクションが見つかりません"

#: kpi/serializers/v2/asset.py:848
#: kpi/tests/api/v2/test_api_collections.py:472
msgid "User cannot update target parent collection"
msgstr "ユーザーは対象の親コレクションをアップデートできません"

#: kpi/serializers/v2/asset_export_settings.py:64
msgid "`export_settings` must contain all the following required keys: {}"
msgstr "`export_settings`は、以下の必須のキー{}を含む必要があります"

#: kpi/serializers/v2/asset_export_settings.py:75
msgid "`export_settings` can contain only the following valid keys: {}"
msgstr "`export_settings`は、以下の有効なキー{}のみを含めることができます｝"

#: kpi/serializers/v2/asset_export_settings.py:87
msgid "`multiple_select` must be either {}"
msgstr "`multiple_select`は{}のいずれかでなくてはなりません"

#: kpi/serializers/v2/asset_export_settings.py:94
msgid "`type` must be either {}"
msgstr "`type`は{}のいずれかでなくてはなりません"

#: kpi/serializers/v2/asset_export_settings.py:104
msgid "`group_sep` must be a non-empty value"
msgstr "`group_sep`は、非空のバリューでなくてはなりません"

#: kpi/serializers/v2/asset_export_settings.py:109
msgid "`lang` for this asset must be either {}"
msgstr "このアセットの`lang`は、{}のいずれかでなくてはなりません"

#: kpi/serializers/v2/asset_export_settings.py:119
#: kpi/serializers/v2/export_task.py:229
msgid "Must be a JSON object"
msgstr "JSONオブジェクトである必要があります"

#: kpi/serializers/v2/asset_export_settings.py:134
#: kpi/serializers/v2/export_task.py:247
msgid "All values in the array must be integers"
msgstr "アレー中のバリューはすべて、整数である必要があります"

#: kpi/serializers/v2/asset_export_settings.py:145
msgid "`fields` must be an array"
msgstr "`fields`はアレーである必要があります"

#: kpi/serializers/v2/asset_export_settings.py:149
msgid "All values in the `fields` array must be strings"
msgstr "`fields`アレー中のバリューはすべて文字列である必要があります"

#: kpi/serializers/v2/asset_file.py:117
msgid "JSON is invalid"
msgstr "JASONは無効です"

#: kpi/serializers/v2/asset_file.py:130
msgid "Invalid content"
msgstr "無効なコンテンツ"

#: kpi/serializers/v2/asset_file.py:143
msgid "No files have been submitted"
msgstr "ファイルは投稿されていません"

#: kpi/serializers/v2/asset_file.py:158
msgid "File already exists"
msgstr "フィルはすでに存在します"

#: kpi/serializers/v2/asset_file.py:208
msgid ""
"You cannot upload media file with two different ways at the same time. "
"Please choose between binary upload, base64 or remote URL."
msgstr "メディアファイルを、二種類の異なった方法で同時にアップロードできません。バイナリーアップロード、ベース64、リモートＵＲＬの中から選んでください。"

#: kpi/serializers/v2/asset_file.py:221
msgid "This field is required"
msgstr "このフィールドは必須です"

#: kpi/serializers/v2/asset_file.py:229
msgid "`redirect_url` is required"
msgstr "`redirect_url`は必須です"

#: kpi/serializers/v2/asset_file.py:239
msgid "`filename` is required"
msgstr "`filename`は必須です"

#: kpi/serializers/v2/asset_file.py:253
msgid "`redirect_url` is invalid"
msgstr "`redirect_url`は無効です"

#: kpi/serializers/v2/asset_file.py:287
msgid "Only `{}` MIME types are allowed"
msgstr "`{}`MIMEタイプのみ許可されます"

#: kpi/serializers/v2/asset_permission_assignment.py:163
msgid "This field is required for the '{}' permission"
msgstr "このフィールドは '{}' パーミッションに必要です"

#: kpi/serializers/v2/asset_permission_assignment.py:177
#: kpi/serializers/v2/asset_permission_assignment.py:182
#: kpi/serializers/v2/asset_permission_assignment.py:188
#: kpi/serializers/v2/asset_permission_assignment.py:497
msgid "Invalid `url`"
msgstr "無効な`url`"

#: kpi/serializers/v2/asset_permission_assignment.py:193
msgid "Invalid `filters`"
msgstr "無効な`filters`"

#: kpi/serializers/v2/asset_permission_assignment.py:210
#, python-brace-format
msgid ""
"{permission} cannot be assigned explicitly to Asset objects of this type."
msgstr "{permission}はこのタイプのアセットオブジェクトに明確に割り当てられることはできません。"

#: kpi/serializers/v2/asset_permission_assignment.py:289
msgid "Not a valid list."
msgstr "有効なリストではありません。"

#: kpi/serializers/v2/asset_permission_assignment.py:490
#: kpi/serializers/v2/asset_permission_assignment.py:493
msgid "Invalid hyperlink - Object does not exist."
msgstr "無効なハイパーリンクです。オブジェクトがありません。"

#: kpi/serializers/v2/data.py:70
msgid "`submission_ids` must only contain integer values"
msgstr "`submission_ids`は整数値のみを含む必要があります"

#: kpi/serializers/v2/data.py:75
msgid "`submission_ids` must contain at least one value"
msgstr "`submission_ids`は一つ以上の値を含む必要があります"

#: kpi/serializers/v2/data.py:83
msgid "`data` is required"
msgstr "`data`は必須です"

#: kpi/serializers/v2/data.py:91
msgid "`validation_status.uid` is required"
msgstr "`validation_status.uid`は必須です"

#: kpi/serializers/v2/export_task.py:123
msgid "Must contain all the following required keys: {}"
msgstr "以下にあげる必須のキー{}すべてを含んでいる必要があります"

#: kpi/serializers/v2/export_task.py:137
msgid "Can contain only the following valid keys: {}"
msgstr "以下の有効なキー{}のみを含めることができます"

#: kpi/serializers/v2/export_task.py:152 kpi/serializers/v2/export_task.py:237
msgid "Must be an array"
msgstr "アレーである必要があります"

#: kpi/serializers/v2/export_task.py:159
msgid "All values in the array must be strings"
msgstr "アレーのバリューはすべて文字列である必要があります"

#: kpi/serializers/v2/export_task.py:169
msgid "Must be a non-empty value"
msgstr "非空のバリューである必要があります"

#: kpi/serializers/v2/export_task.py:182
msgid "For this asset must be either {}"
msgstr "このアセットに対しては、{}のいずれかである必要があります"

#: kpi/serializers/v2/export_task.py:194 kpi/serializers/v2/export_task.py:258
msgid "Must be either {}"
msgstr "{}のいずれかでなくてはなりません"

#: kpi/serializers/v2/export_task.py:204
msgid "The asset must be deployed."
msgstr "アセットは配置される必要があります。"

#: kpi/serializers/v2/export_task.py:221
msgid "The export name must be a string."
msgstr "エクスポートの名前は文字列である必要があります。"

#: kpi/serializers/v2/paired_data.py:87
msgid "Source cannot be changed"
msgstr "ソースを変更できません"

#: kpi/serializers/v2/paired_data.py:93
#, python-brace-format
msgid "Data sharing for `{source_uid}` is not enabled"
msgstr "`{source_uid}`とのシェアリングはできません"

#: kpi/serializers/v2/paired_data.py:105
#, python-brace-format
msgid "Pairing data with `{source_uid}` is not allowed"
msgstr "`{source_uid}`とのペアリングは許可されません"

#: kpi/serializers/v2/paired_data.py:110
#, python-brace-format
msgid "Source `{source}` is already paired"
msgstr "ソース`{source}`は既にペアリングされています"

#: kpi/serializers/v2/paired_data.py:152
#, python-brace-format
msgid "Some fields are invalid, choices are: `{source_fields}`"
msgstr "無効なフィールドがあります。選択は`{source_fields}`"

#: kpi/serializers/v2/paired_data.py:177
msgid "Only letters, numbers and `-` are allowed"
msgstr "文字、数字と`-`のみ許可されています"

#: kpi/serializers/v2/paired_data.py:184
msgid "Extension must be `xml`"
msgstr "拡張子は、`xml`でなくてはなりません"

#: kpi/serializers/v2/paired_data.py:217
#, python-brace-format
msgid "`{basename}` is already used"
msgstr "`{basename}`は既に使われています"

#: kpi/serializers/v2/user_asset_subscription.py:47
#, python-brace-format
msgid "Invalid asset type. Only `{asset_type}` is allowed"
msgstr "無効なアセットのタイプです。`{asset_type}`のみ許可されています"

#: kpi/templates/modern_browsers.html:3 kpi/templates/modern_browsers.html:5
msgid "Modern Browsers"
msgstr "モダンブラウザ"

#: kpi/templates/modern_browsers.html:8
msgid ""
"Ouch! Sorry, you were probably sent here because your browser is outdated or"
" perhaps it has its key features disabled by using \"private browsing\" "
"mode."
msgstr "申し訳ございません。ご使用のブラウザが古いか、「プライベートブラウジング」モードで主要な機能を無効にしているため、ここに送られたと思われます。"

#: kpi/templates/modern_browsers.html:12
msgid ""
"If you are using an outdated browser, we recommend upgrading to a recent "
"version of one of the following excellent modern browsers:"
msgstr "古いブラウザをお使いの場合は、以下の優れたモダンブラウザの最新版にアップグレードすることをお勧めします："

#: kpi/templates/modern_browsers.html:17
msgid "Firefox download page"
msgstr "Firefoxダウンロードページ"

#: kpi/templates/modern_browsers.html:18
msgid "Firefox"
msgstr "Firefox"

#: kpi/templates/modern_browsers.html:22
msgid "Chrome download page"
msgstr "Chromeダウンロードページ"

#: kpi/templates/modern_browsers.html:23
msgid "Chrome"
msgstr "Chrome"

#: kpi/templates/modern_browsers.html:27
msgid "Edge download page"
msgstr "Edgeダウンロードページ"

#: kpi/templates/modern_browsers.html:28
msgid "Edge"
msgstr "Edge"

#: kpi/templates/modern_browsers.html:32
msgid "Opera download page"
msgstr "Operaダウンロードページ"

#: kpi/templates/modern_browsers.html:33
msgid "Opera"
msgstr "Opera"

#: kpi/tests/test_registration.py:71
msgid "This email domain is not allowed to create an account"
msgstr "このメールドメインではアカウントの作成が許可されていません"

#: kpi/utils/two_database_configuration_checker.py:41
msgid "Exactly two databases must be configured"
msgstr "二つのデータベースは厳密に設定されなければなりません"

#: kpi/utils/two_database_configuration_checker.py:42
msgid ""
"KPI and KoBoCAT must each have their own databases. Configure the KPI "
"database as 'default' and the KoBoCAT database as 'kobocat'."
msgstr "KＰＩと KｏBｏCＡＴには、それぞれ固有のデータベースがあります。KＰＩデータベースを「デフォルト」に、KｏBｏCＡＴを「kobocat」に設定してください。"

#: kpi/utils/two_database_configuration_checker.py:63
msgid "KPI may not share a database with KoBoCAT"
msgstr "KPIは、KoBoCATとデータベースを共有できません"

#: kpi/utils/two_database_configuration_checker.py:64
msgid "KPI and KoBoCAT must each have their own databases."
msgstr "KPIとKoBoCATは、それぞれ固有のデータベースがある必要があります。"

#: kpi/utils/two_database_configuration_checker.py:133
msgid "Incomplete migration from shared-database installation"
msgstr "共有データベースのインストールからの不完全な移行"

#: kpi/utils/two_database_configuration_checker.py:134
msgid ""
"The KoBoCAT database was originally shared by KPI, but the KPI tables were "
"not copied from that shared database to the new, KPI-only database."
msgstr "KoBoCATデータベースは本来はＫＰＩと共有していましたが、ＫＰＩテーブルは共有データベースから新しいKPI専用データベースにはコピーされませんでした。"

#: kpi/views/current_user.py:107
msgid "Invalid confirmation"
msgstr "無効な確認"

#: kpi/views/v2/asset_export_settings.py:228
msgid "Only the following formats are available: ##format list##"
msgstr "下記のフォーマットのみ有効です: ##format list##"

#: kpi/views/v2/asset_permission_assignment.py:211
msgid "Source and destination objects don't seem to have the same type"
msgstr "ソースと目標オブジェクトは、同じタイプを持っていないようです"

#: kpi/views/v2/asset_permission_assignment.py:240
msgid "Owner's permissions cannot be deleted"
msgstr "所有者のパミッションは削除できません"

#: kpi/views/v2/attachment.py:73
msgid "`xpath` query parameter is required"
msgstr "`xpath`クエリのパラメーターは必須です"

#: kpi/views/v2/attachment.py:94
msgid "Invalid XPath syntax"
msgstr "無効なXPath構文"

#: kpi/views/v2/attachment.py:98
msgid "The path could not be found in the submission"
msgstr "投稿にパスは見つけられませんでした"

#: kpi/views/v2/attachment.py:107
msgid "The error occurred during conversion"
msgstr "コンバージョン中にエラーが発生しました"

#: kpi/views/v2/attachment.py:111
msgid "Conversion is not supported for {}"
msgstr "コンバージョンは{}をサポートしません"

#: kpi/views/v2/data.py:318
msgid "The specified asset has not been deployed"
msgstr "指定されたアセットは配置されていません"

#: kpi/views/v2/data.py:622
msgid "A positive integer is required"
msgstr "正の整数が必要です"
